Not to use openshift.io/cluster-monitoring label on user-defined projects (#56853)
openshift/openshift-docs: modules/monitoring-enabling-monitoring-for-user-defined-projects.adoc

In {product-title} {product-version} you must remove any custom Prometheus instances before enabling monitoring for user-defined projects.
====

> [Note]

Reviewer: ```suggestion
[NOTE]
```
But consider using IMPORTANT.

====

> Label `openshift.io/cluster-monitoring` is reserved for platform specific projects. You must not use the label `openshift.io/cluster-monitoring` on any user-defined projects.

Reviewer: ```suggestion
The `openshift.io/cluster-monitoring` label is reserved for platform-specific projects. Do not use the label `openshift.io/cluster-monitoring` on any user-defined projects.
```
